ChaosMod/Effects/db/Misc/MiscSelfieTime.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+	Effect by OnlyRealNubs
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	DESTROY_MOBILE_PHONE();

```

Comment:
This native causes soft locks during important missions that rely on phone interaction such as The Merryweather Heist (Offshore Variant). Please avoid using this native and find a way to wait for the phone call to be over.

Reply:
That wasn't supposed to be there. I might of typed it on accident. I will immediately remove it.

Reply:
That whole file was accidentally included.